.github/workflows/tests.yml

```diff
@@ -85,8 +85,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        bazel_version: [6.5.0, 7.1.0]
-        sandbox: [true, false]
```

Comment:
Same here

Reply:
done!

Reply:
Do we want to keep [7.1.0] only?

Reply:
Yeah we can keep it at 7.1 unless we se a need for a new Bazel 7 version